### PR TITLE
preliminary version of community award post

### PR DESCRIPTION
**Pull request by ChristinaLK**
To do: 
- [x] Review wording (esp. of opening) @k8hertweck @raynamharris @mkuzak @smcclatchy @karinlag
- [x] Confirm who can access the nominations email address + who will be monitoring

### CODE REVIEW

**Comment by k8hertweck** [comment]
Maybe rephrase the first question as "Is there a Software Carpentry community member you've noticed working extra hard to help our organization?"

Or at least start the question like that, to focus on community members in particular. 

Perhaps also note you can see previous awardees on the webpage link?

Thanks @ChristinaLK !

**Comment by ChristinaLK** [comment]
I added a tagline -- does our blog support that option?  Let me know + I can move it to the body of the post.  

Otherwise, ready to merge.
